db/rdbms/src/main/java/io/camunda/db/rdbms/read/service/ProcessDefinitionReader.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.db.rdbms.read.service;
+
+import io.camunda.db.rdbms.sql.ProcessDefinitionMapper;
+import io.camunda.db.rdbms.write.domain.ProcessDefinitionDbModel;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class ProcessDefinitionReader {
+
+  private final ProcessDefinitionMapper processDefinitionMapper;
+  private final ConcurrentHashMap<Long, ProcessDefinitionDbModel> cache = new ConcurrentHashMap<>();
```

Comment:
Actually no, there is no TTL or remove logic. Also this could be considered as leftover from previous iterations, currently we fetch the processName via LEFT JOIN directly in the SQL because it's far easier.

I will remove the cache.